common/projectconfig/projectconfig.go

```diff
@@ -92,6 +97,21 @@ func loadFile(path string) (Config, error) {
 	return config, nil
 }
 
+func CreateAndSave(config Config) error {
```

Comment:
I'm not sure what this adds over `Save()`, which will always recreate the file?

Reply:
Oh thanks for catching that - I only made this to call `GetDefaultConfigPath()`, so I just moved that up to the resolver instead so it could call `Save()` directly.